src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -329,11 +331,13 @@ class CommandRequestImpl : public CommandImpl,
       const mobile_apis::AppHMIType::eType app_hmi_type) const;
 
   /**
-   * @brief Adds information to result message "info" param in case of GENERIC_ERROR
+   * @brief Adds information to result message "info" param in case of
+      GENERIC_ERROR
```

Comment:
@pold500 I suggest to put "in case of GENERIC_ERROR" in round brackets and please fix layout for this sentence.